core/types/calls.cc

```diff
@@ -35,6 +35,14 @@ bool allComponentsPresent(DispatchResult &res) {
     }
     return allComponentsPresent(*res.secondary);
 }
+
+inline bool isTyped(const TypePtr type) {
+    return type && !type.isUntyped();
```

Comment:
Why is the nullptr check required?

Reply:
I can't recall the exact scenario, but I got a crash running the change over pay-server. I've rerun in now without nullptr check, and it didn't crash this time. It should be safe to remove it know